doc/admin/pool_operations.md

```diff
@@ -301,6 +301,22 @@ $ dmg pool reintegrate --pool=${puuid} --rank=5 --target-idx=0,1
 Support for online target addition and automatic space rebalancing is
 planned for DAOS v1.4 and will be documented here once available.
 
+An operator can choose to extend a pool to include ranks not currently in the pool.
+This will automatically trigger a server rebalance operation where objects within the extended
+pool will be rebalanced across the new storage.
+
+```
+$ dmg pool extend --pool=${puuid} --ranks=${rank1},${randk2}...
+```
+
+The pool extend command accepts 2 required parameters:
+
+* The pool UUID of the pool to be extended.
+* A comma separated list of ranks to include in the pool.
```

Comment:
Suggestion: "server ranks" might be clearer to someone who is still becoming acquainted with DAOS.